bin/spark-submit2.cmd

```diff
@@ -1,78 +0,0 @@
-@echo off
-
```

Comment:
wait, what will `bin/spark-submit.cmd` call then? I think we still need this file